specification/communication/Communication.Messages/examples/2024-02-01/Template_List.json

```diff
@@ -0,0 +1,37 @@
+{
+  "title": "Gets a list of templates",
+  "operationId": "TemplateOperations_List",
+  "parameters": {
+    "api-version": "2024-02-01",
+    "content-type": "application/json",
+    "channelId": "0f0eb78e-a576-1dfc-b414-0f0756de3338",
+    "endpoint": "https://my-resource.communication.azure.com"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "sample_shipping_confirmation",
+            "language": "en_US",
+            "channelType": "whatsApp",
+            "status": "approved",
+            "whatsApp": {
+              "content": [
+                {
+                  "type": "BODY",
```

Comment:
This example doesn't match the model anymore, it seems?

Reply:
updated